templates/static/js/src/demobox/components/LiveDemoBox/LiveDemoBox.tsx

```diff
@@ -83,41 +82,38 @@ const LiveDemoBox = () => {
     }
   };
 
-  const url = `/docs/examples/patterns/notifications/toast?type=${selectValue.type}&style=${selectValue.style}&actions=${selectValue.actions}&dismiss=${selectValue.dismiss}&timestamp=${selectValue.timestamp}`;
-
+  const constructUrl = () => {
+    let url = `${jsonUrl}?`;
+    const arrayOfKeys = Object.keys(selectValue);
+    arrayOfKeys.forEach((key) => {
+      url += `${key}=${selectValue[key]}&`;
+    });
+    return url;
+  };
+  const url = constructUrl();
+  const dropdownOptions = configValues && Object.keys(configValues.dropdown);
   return (
     <section className="p-strip--light">
       {configValues && configValues.dropdown && configValues.switch && (
         <form>
           <div className="row" style={{ gridGap: 0 }}>
-            <div className="col-2">
-              <Select
-                id="type"
-                label="Type"
-                name="type"
-                onChange={handleChange}
-                options={configValues.dropdown.type}
-              />
-            </div>
-            <div className="col-2">
-              <Select
-                id="theme"
-                label="Theme"
-                name="theme"
-                onChange={handleChange}
-                options={configValues.dropdown.theme}
-                disabled
-              />
-            </div>
-            <div className="col-2">
-              <Select
-                id="style"
-                label="Style"
-                name="style"
-                onChange={handleChange}
-                options={configValues.dropdown.style}
-              />
-            </div>
+            {dropdownOptions?.map((dropdownOption) => {
+              const optionValue = configValues?.dropdown[dropdownOption];
+              return (
+                <div className="col-2" key={dropdownOption}>
+                  <Select
+                    id={dropdownOption}
+                    label={
+                      dropdownOption[0].toUpperCase() +
+                      dropdownOption.substring(1)
+                    }
+                    name={dropdownOption}
+                    onChange={handleChange}
+                    options={optionValue}
+                  />
+                </div>
+              );
+            })}
```

Comment:
I tried to render the selects dynamically but the code looks a bit hacky. It might need to be updated